src/Lean/Linter/ConstructorAsVariable.lean

```diff
@@ -53,7 +58,7 @@ def constructorNameAsVariable : Linter where
               if !cmdStxRange.contains range.start || ldecl.userName.hasMacroScopes then return
               let opts := ci.options
               -- we have to check for the option again here because it can be set locally
-              if !linter.constructorNameAsVariable.get opts then return
+              if !getLinterValue linter.constructorNameAsVariable opts then return
```

Comment:
Any reason you use getLinterConstructorNameAsVariable at the top of the function but inline it here? I'd lean towards not having `getLinterConstructorNameAsVariable` at all, but if you really want it then it would be good to use consistently.